crates/ide-assists/src/handlers/add_missing_impl_members.rs

```diff
@@ -2363,4 +2363,33 @@ impl other_file_2::Trait for MyStruct {
 }"#,
         );
     }
+
+    #[test]
+    fn unsafeness_observed() {
```

Comment:
Why is **this** assist affected? In fact I'd prefer if it remains as-is, changing the trait header for "implement missing members" seems surprising.